flashlight/fl/test/CMakeLists.txt

```diff
@@ -8,7 +8,8 @@ build_test(SRC ${DIR}/common/DynamicBenchmarkTest.cpp LIBS ${LIBS})
 build_test(SRC ${DIR}/common/HistogramTest.cpp LIBS ${LIBS})
 build_test(SRC ${DIR}/common/LoggingTest.cpp LIBS ${LIBS})
 build_test(SRC ${DIR}/common/SerializationTest.cpp LIBS ${LIBS})
-build_test(SRC ${DIR}/common/SequentialBuilderTest.cpp LIBS ${LIBS})
+build_test(SRC ${DIR}/common/SequentialBuilderTest.cpp LIBS ${LIBS}
```

Comment:
Ditto with my previous comment on the PR where you moved this. I'm slightly impartial to this belonging in `pkg/common`

Reply:
now in pkg/runtime